packages/hoppscotch-common/src/components/environments/my/index.vue

```diff
@@ -32,12 +32,17 @@
       :environment="environment"
       @edit-environment="editEnvironment(index)"
     />
-    <div v-if="!environments.length" class="p-5 space-y-25 flex flex-col">
+    <HoppSmartPlaceholder
+      v-if="!environments.length"
+      :src="`/images/states/${colorMode.value}/blockchain.svg`"
+      :alt="`${t('empty.environments')}`"
+      :text="t('empty.environments')"
+    >
       <div class="flex flex-col items-center space-y-4">
         <span class="text-secondaryLight text-center">
           {{ t("environment.import_or_create") }}
         </span>
-        <div class="flex gap-4 flex-col items-center">
+        <div class="flex gap-4 flex-col items-stretch">
```

Comment:
Is the `items-stretch` class necessary here?

Reply:
Yep, making the button to stretch the parent width is intentional.